.aws/src/config/index.ts

```diff
@@ -37,6 +37,10 @@ export const config = {
     repository: 'pocket/curated-corpus-api',
     branch,
   },
+  // The name of the Event Bus to publish events to (per account)
```

Comment:
we need  an `iam` to publish from `curatedCorpusApi` to the event bus. 
https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-bus-perms.html#eb-event-bus-example-policy-cross-account

also can we test this in dev to make sure the integration from curatedCorpusApi -> eventBus works

Reply:
as far as I can tell, no special permission is required to publish to the default event bus for a given AWS account

Reply:
I was wrong, tested with updated permissions